wiki/osu!_Community_Meetings/id.md

```diff
@@ -0,0 +1,37 @@
+# Pertemuan komunitas osu!
+
+**Pertemuan komunitas osu!** adalah panel diskusi yang diadakan oleh [tim inti osu!](/wiki/People/The_Team) setiap dua minggu sekali. Tujuan utama dari pertemuan-pertemuan tersebut adalah untuk memberikan kesempatan untuk seluruh pihak untuk berbicara secara langsung dengan pengembang dan orang-orang yang bertanggung jawab dalam mengatur komunitas untuk mengangkat masalah-masalah untuk didiskusikan atau pertimbangan lebih lanjut.
+
+[Pada masa yang sangat lampau osu!](https://twitter.com/ppy/status/1169256824052170755), dulunya tim inti osu! rutin mengadakan pertemuan komunitas di mana seluruh orang dapat berbicara. Dengan ukuran permainan saat ini, hal ini tidak dapat dicapai, tetapi dengan menggunakan sistem antrian Discord yang dapat mengurutkan pertanyaan-pertanyaan yang dapat dibahas satu per satu. Jika kamu tertarik untuk bertanya, mohon pastikan kamu telah membaca FAQ di bawah ini untuk mengecek apakah pertanyaanmu sudah pernah ditanyakan dan dijawab!
+
+Saat ini, pertemuan komunitas osu! diketuai oleh [peppy](https://osu.ppy.sh/users/2), [Ephemeral](https://osu.ppy.sh/users/102335), dan [smoogipoo](https://osu.ppy.sh/users/1040328).
+
+Pertemuan komunitas osu! berikutnya akan diadakan pada **Minggu, 28 November 2021 pukul 01:00 UTC**. Pertemuan diadakan setiap dua minggu dengan pengingat yang diumumkan melalui [akun Twitter peppy](https://twitter.com/ppy) sehari sebelumnya.
+
+## Cara berpartisipasi
+
+Pertemuan komunitas osu! saat ini diadakan di Discord Stage pada [server Discord osu!dev](https://discord.gg/ppy) dan disiarkan pada [kanal Twitch peppy](https://www.twitch.tv/ppy). Setiap pertemuan saat ini dijadwalkan untuk berlangsung selama satu jam dan dimulai setiap hari Minggu pukul 01.00 UTC.
```

Comment:
typo in the meeting time, i think

Reply:
not all languages use a period between hours and minutes

Reply:
paragraph above this says `pukul 01:00 UTC` and here it's `pukul 01.00 UTC` so i comment. resolve if it's already correct though

Reply:
ah we use dot to separate hour, minute, and second in time. [Pedoman Umum Ejaan Bahasa Indonesia (General Guidelines for Indonesian Spelling)](https://puebi.readthedocs.io/en/latest/tanda-baca/tanda-titik/#a-tanda-titik)